validation/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -54,7 +54,8 @@ object PredicateCommon {
   def numeric[T]: Predicate[T] = new Predicate[T] {
     override def validate: (T) => Boolean = _.asInstanceOf[AnyRef] match {
       case n: Number => true
-      case _ => throw new NotImplementedError("'numeric' doesn't handle string (or other) values yet")
+      case s: String => Try(s.toDouble).isSuccess
```

Comment:
No, it was something I wrote for an approach that was later dropped.  I figured it might be useful, but I can just close the ticket if this looks unnecessary